pkg/resources/testdata/TestAcc_Task/basic/test.tf

```diff
@@ -0,0 +1,82 @@
+resource "snowflake_task" "test" {
+  name          = var.name
+  database      = var.database
+  schema        = var.schema
+  started       = var.started
+  sql_statement = var.sql_statement
+
+  # Optionals
+  warehouse                   = var.warehouse
+  config                      = var.config
+  allow_overlapping_execution = var.allow_overlapping_execution
+  error_integration           = var.error_integration
+  when                        = var.when
+  comment                     = var.comment
+
+  dynamic "schedule" {
+    for_each = [for element in [var.schedule] : element if element != null]
+    content {
+      minutes    = lookup(var.schedule, "minutes", null)
+      using_cron = lookup(var.schedule, "cron", null)
+    }
+  }
+
+  # Parameters
+  suspend_task_after_num_failures               = var.suspend_task_after_num_failures
```

Comment:
bleh, we have to implement missing parts for the model gen and get rid of such configs